sonar-project.properties

```diff
@@ -2,7 +2,7 @@ sonar.projectKey = prometheus-juju-backup-all-exporter
 sonar.organization = bootstack-devops
 sonar.python.version = 3
 sonar.python.coverage.reportPaths=/report/coverage.xml
```

Comment:
sonar.python.coverage.reportPaths must be relative

Reply:
like `./report/coverage.xml` ?

Reply:
👌🏼